finetuner/__init__.py

```diff
@@ -67,7 +67,7 @@ def _build_name_stub_map() -> Dict[str, model_stub.ModelStubType]:
 
 def list_models() -> List[str]:
     """List available models."""
-    return [name for name in list_model_classes()]
+    return [stub.display_name for stub in list_model_classes().values()]
```

Comment:
I think you also need to adjust `list_model_options()` in this case

Reply:
I have tested both and they work